lib/std/Random/Isaac64.zig

```diff
@@ -150,24 +150,19 @@ fn seed(self: *Isaac64, init_s: u64, comptime rounds: usize) void {
 }
 
 pub fn fill(self: *Isaac64, buf: []u8) void {
-    var i: usize = 0;
     const aligned_len = buf.len - (buf.len & 7);
 
-    // Fill complete 64-byte segments
-    while (i < aligned_len) : (i += 8) {
-        var n = self.next();
-        comptime var j: usize = 0;
-        inline while (j < 8) : (j += 1) {
-            buf[i + j] = @as(u8, @truncate(n));
-            n >>= 8;
-        }
+    // Complete 8 byte segments.
+    const buf64 = std.mem.bytesAsSlice(u64, buf[0..aligned_len]);
```

Comment:
Do you think we'd have more luck picking an optimal size for more platforms by using `usize` instead of `u64`?

Reply:
Ah nevermind, the prng types seem to be returning `u64`, ignore me :)